Valet.xcodeproj/project.pbxproj

```diff
@@ -1759,7 +1777,7 @@
 				CURRENT_PROJECT_VERSION = 1;
 				DEBUG_INFORMATION_FORMAT = dwarf;
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 9XUJ7M53NG;
```

Comment:
Does this need to specify `DEVELOPMENT_TEAM`/`CODE_SIGN_IDENTITY` here?  Probably better to leave these blank.

Reply:
I've needed the dev team in the past to get unit tests to work in a signed environment (I think). I'll try removing. If it passes, then yeah, let's remove :)